src/main/resources/static/css/buttonOverrides.css

```diff
@@ -1,3 +1,25 @@
+.button_disabled
+{
+    display: inline-block;
+    margin-bottom: 0;
+    font-weight: 400;
+    text-align: center;
+    vertical-align: middle;
+    cursor: pointer;
+    background-image: none;
+    border: 1px solid transparent;
+    padding: 6px 12px;
+    font-size: 14px;
+    line-height: 1.4;
+    border-radius: 4px;
+    -webkit-user-select: none;
+
+    color: #fff;
+    background-color: #D3D3D3;
+    border-color: #999999;
+
+}
+
 .hovering-purple-button
 {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the opening curly brace to the previous line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=css%3Aformatting)